app/Models/QueryBuilders/SearchFormQueryBuilder.php

```diff
@@ -136,6 +170,26 @@ public function buildQuery(): array
             ];
         }
 
+        if ((isset($this->startDate) && !empty($this->startDate)) || (isset($this->endDate) && !empty($this->endDate))) {
+            $dateRange = [];
+
+            if (isset($this->startDate) && !empty($this->startDate)) {
+                $carbonStartDate = new Carbon($this->startDate);
+                $dateRange['gte'] = $carbonStartDate->toAtomString();
+            }
+
+            if (isset($this->endDate) && !empty($this->endDate)) {
```

Comment:
Same here.

Reply:
"isset" checks if user want to search for date
"empty" checks if date is not empty string ''.